# Pick up easybuild build speech_tools config
# SEP 9th 2020 by B. Hajgato (UGent)
--- config/config.in.org	2017-09-04 17:54:08.000000000 +0200
+++ config/config.in	2020-09-08 11:27:08.819754926 +0200
@@ -10,7 +10,7 @@
 ###########################################################################
 ## Which speech tools to use
 
-EST=$(TOP)/../speech_tools
+EST=${EBROOTSPEECH_TOOLS}
 
 ###########################################################################
 ## Where the festival tree will be installed.
